pkg/provider/azure.go

```diff
@@ -173,35 +176,13 @@ type Config struct {
 	// the cloudprovider will try to add all nodes to a single backend pool which is forbidden.
 	// In other words, if you use multiple agent pools (scale sets), you MUST set this field.
 	PrimaryScaleSetName string `json:"primaryScaleSetName,omitempty" yaml:"primaryScaleSetName,omitempty"`
-	// Enable exponential backoff to manage resource request retries
-	CloudProviderBackoff bool `json:"cloudProviderBackoff,omitempty" yaml:"cloudProviderBackoff,omitempty"`
```

Comment:
is this `maligned` rule required? this change would move such correlated configuration options to different places and would make future changes harder. Could we keep it disabled?

Reply:
it's not "required" but it's best practice. It detects structs that would take less memory if their fields were sorted.

For example in this case 
```
pkg/provider/azure.go:131:13: struct of size 800 bytes could be of size 776 bytes (maligned)
```

Reply:
OK, let's add this then.